autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -650,6 +665,116 @@ def test_scale_up_down_on_rcv3_and_clb(self):
                           'count of {1}'.format(num_clb_nodes_after_scale,
                                                 self.min_servers))
 
+    @tags(speed='slow', type='rcv3', rcv3_mimic='pass')
+    def test_scale_down_after_manual_remove_node(self):
+        """
+        Verify that the RCV3 group can successfully scale down after a server
+        has been removed from the loadbalancer pool outside of autoscale.
+        """
+
+        # Capture a list of the node_ids of all nodes on the pool before doing
+        # anything.
+        initial_node_list = [n.id for n in self.rcv3_client.get_nodes_on_pool(
```

Comment:
This can be a separate function:
```python
def get_node_ids_on_pool(pool_id):
   return [n.id for n in self.rcv3_client.get_nodes_on_pool(pool_id)]
```
as it is being used in multiple places